CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 cmake_minimum_required(VERSION 2.8)
+set(CMAKE_LEGACY_CYGWIN_WIN32 0) # Remove when CMake >= 2.8.4 is required
```

Comment:
I don't mind bumping minimal required CMake version to 2.8.4 instead (which is 5 years old at this point).